chemicaltoolbox/data_source/get_pubchem/get_pubchem_as_smiles.xml

```diff
@@ -28,7 +28,6 @@
 ]]>
     </command>
     <inputs>
-        <param name="infile" type="select" display="radio" label="Load all pubchem files and convert them to canonical smiles." />
```

Comment:
Oh is this working today? No input at all? Cool!

Reply:
It works with `planemo s` at least.